slice-core/src/main/java/com/cognifide/slice/core/internal/injector/InjectorHierarchy.java

```diff
@@ -200,7 +202,7 @@ private Injector createInjector(InjectorConfig config) {
 			}
 			return injector;
 		} catch (CreationException e) {
-			LOG.error("Can't create injector " + config.getName(), e);
```

Comment:
Restore this LOG. Otherwise we'll lose the information why the injector can't be created.